src/main/java/com/hubspot/jinjava/Jinjava.java

```diff
@@ -123,6 +127,10 @@ public Context getGlobalContext() {
     return globalContext;
   }
 
+  public Context getGlobalContextReadOnly() {
```

Comment:
```suggestion
  public Context getGlobalContextCopy() {
```

Reply:
I used to use `getGlobalContextCopy`, thought the readOnly one is better :).